src/main/scala/org/neo4j/spark/Neo4jDataFrame.scala

```diff
@@ -16,25 +16,51 @@ import scala.collection.JavaConverters._
 object Neo4jDataFrame {
 
   def mergeEdgeList(sc: SparkContext, dataFrame: DataFrame, source: (String,Seq[String]), relationship: (String,Seq[String]), target: (String,Seq[String])): Unit = {
```

Comment:
default value for renameMap perhaps better call it "renamedColumns"

Reply:
done